modules/odc-creating-a-binding-connection-between-components.adoc

```diff
@@ -1,36 +1,35 @@
 // Module included in the following assemblies:
 //
 // * applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc
+
 :_content-type: PROCEDURE
 [id="odc-creating-a-binding-connection-between-components_{context}"]
 = Creating a binding connection between components
 
-You can establish a binding connection with Operator-backed components.
+You can create a binding connection between components, as demonstrated in the following example that uses a PostgresSQL database service and a Spring PetClinic sample application.
 
```

Comment:
Consider moving the following lines up here and rewriting them as a conceptual, more general introduction into this procedure:
```Suggestion
A binding connection is create using a `ServiceBinding` resource. In this example:

[source,YAML]
----
apiVersion: binding.operators.coreos.com/v1alpha1
kind: ServiceBinding
metadata:
    name: spring-petclinic-pgcluster
    namespace: my-petclinic
spec:
    services:
    - group: postgres-operator.crunchydata.com
      version: v1beta1
      kind: PostgresCluster
      name: hippo
    application:
      name: spring-petclinic
      group: apps
      version: v1
      resource: deployments
----

[TIP]
====
As an alternative to the Topology view, you can go to *+Add* -> *YAML* -> *Import YAML* and enter this `ServiceBinding` in the YAML editor.
====
```

Reply:
Fixed.